### PR TITLE
Minor fixes/updates

### PR DESCRIPTION
**Pull request by CasperWA**
Running through the sections I am chair/co-chair on, and found some very minor fixes/updates.

~Missing going through `"Working with data and querying your results"`.~

Minor fixes:
- Write `Quantum ESPRESSO` all over.
- Stay within the Aïda universe with example names (instead of Disney).

### CODE REVIEW

**Comment by chrisjsewell** [comment]
what the heck is radames? mickeymouse is a lot better 😉

**Comment by CasperWA** [comment]
> what the heck is radames? mickeymouse is a lot better 😉

https://en.wikipedia.org/wiki/Aida

He's the "Romeo" to Aïda's "Juliet" ... 😉

File: docs/pages/2020_Intro_Week/sections/data.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ User
 
    The ``-a`` and ``-A`` flags we used above ensure that groups for all type strings and users are shown respectively.
 
-We can then inspect a groups contents, by the label (if it is unique) or the PK:
+We can then inspect a group's contents by its label (if it is unique) or the PK:
```
Comment (giovannipizzi) [inline docs/pages/2020_Intro_Week/sections/data.rst]: `a group's content` or `the group's contents`? Also, not sure if we really need the `'s`.
My choice would be simply `the group content` or even better `the content of a group`

**Review by giovannipizzi — COMMENTED** [review]
I like the radames change, even if I'm not sure we should merge it now, people who are there and refresh in group A will get very confused... apart from this, I have another comment.

Either we wait for the swap of mickeymouse and we merge this in a couple of days, or we change that in a different PR

File: docs/pages/2020_Intro_Week/sections/data.rst (hunk 1 above)
Comment (CasperWA) [reply docs/pages/2020_Intro_Week/sections/data.rst]: Right. So `the content of a group` is equivalent to `a group's content`. But it's also "better English" to not use many apostrophes... I will change it to the latter and also reevaluate whether this section is talking about showing the content of a specific group or of groups in general.

**Comment by CasperWA** [comment]
> I like the radames change, even if I'm not sure we should merge it now, people who are there and refresh in group A will get very confused... apart from this, I have another comment.
> 
> Either we wait for the swap of mickeymouse and we merge this in a couple of days, or we change that in a different PR

I don't think it's a change that will confuse very many - unless they are in the middle of doing it of course.

**Comment by giovannipizzi** [comment]
There is a conflict! :-D

File: docs/pages/2020_Intro_Week/sections/running.rst [hunk 2]
```diff
@@ -294,7 +294,7 @@ Define a set of input parameters for Quantum ESPRESSO, preparing a dictionary of
        ...:     'SYSTEM': {
        ...:         'ecutwfc': 30.,
        ...:         'ecutrho': 200.,
-       ...:         'mickeymouse': 240.,
+       ...:         'mickeymouseouse': 240.,
```
Comment (giovannipizzi) [inline docs/pages/2020_Intro_Week/sections/running.rst]: mickeymouseouse?

Comment (CasperWA) [reply docs/pages/2020_Intro_Week/sections/running.rst]: Bad search and replace - don't know what happened.

**Pull request merged**